definitions/variable/emissions/emissions.yaml

```diff
@@ -251,18 +266,17 @@
     description: Emissions of {Level-2 Species} from waste (IPCC category 6)
     unit: "{Level-2 Species}"
 
-- Emissions|{Level-2 Species}|Fossil Fuel Fires:
-    description: Emissions of {Level-2 Species} from fossil fuel fires (IPCC category 7)
-    unit: "{Level-2 Species}"
-
 - Emissions|{Level-2 Species}|Other:
-    description: Emissions of {Level-2 Species} from other sources
+    description: Emissions of {Level-2 Species} from other sources, (IPCC category 7, including fossil fuel fires)
     unit: "{Level-2 Species}"
 
-- Emissions|{Level-3 Species}|Capture and Removal:
-    description: Capture and removal of atmospheric {Level-3 Species} using net-negative
-      technologies that are not directly linked to an emissions source,
-      e.g., direct air capture (DAC) or enhanced weathering
+- Emissions|{Level-3 Species}|Other Capture and Removal:
+    description: Capture and removal of atmospheric {Level-3 Species} using other net-negative 
+      technologies that are not reported under other variables as they are not directly linked 
+      to an emissions source, including direct air carbon capture and storage (DACCS) or enhanced 
+      weathering (EW), as well as durable wood products in building elements, mineral products, 
+      ocean fertilization, biomass sinking, ocean alkalinity enhancement (OAE), direct ocean capture, 
+      and other methods
```

Comment:
I think it would be ideal to make this slightly clearer in terms of where the carbon is going, specifically does it go into some permanent storage, the land pool or the ocean (or somewhere else). Maybe that means we just need to add components?

Reply:
There's a detailed "Carbon Removal|..." variable tree for that. This variable just makes the Emissions-tree add up.

Reply:
Ok. @jkikstra this means we'll have to be a bit more careful than we have previously in how we get the input variables for simple climate models. I think we'll basically have to use the "Emissions|*" tree for most things, but then jump into the "Carbon Removal|*" (and maybe the "Gross Emissions|*") trees to work out the details of where the fluxes actually should go.

Reply:
Agreed - so are we OK with keeping this as is? 
This is just meant just such that people put CDR methods under the same "Emissions|CO2|*" variable. 
At the moment I don't think we need to specify these pools, but if you think it would be good to have this here in addition, then maybe you can create an issue to see if we want to add it later. But I think it would be duplicating the Carbon Removal variable too much.